code/modules/busy_space/organizations.dm

```diff
@@ -0,0 +1,461 @@
+	for newly tested posibrains to remain with the company."
+	history = "" // To be written someday.
+	work = "research giant"
+	headquarters = "Earth"
+	motto = ""
+
+	ship_prefixes = list("NSV" = "exploration", "NTV" = "hauling", "NDV" = "patrol", "NRV" = "emergency response")
+	// Note that the current station being used will be pruned from this list upon being instantiated
+	destination_names = list(
+		"NSS Exodus in Nyx",
+		"NCS Northern Star in Vir",
+		"NCS Southern Cross in Vir",
+		"NDV Icarus in Nyx",
+		"an asteroid orbiting Kara",
+		"an asteroid orbiting Rota",
+		)
+
+/datum/lore/organization/tsc/nanotrasen/New()
+	..()
+	spawn(1) // BYOND shenanigans means using_map is not initialized yet.  Wait a tick.
+		// Get rid of the current map from the list, so ships flying in don't say they're coming to the current map.
+		var/string_to_test = "[using_map.station_name] in [using_map.starsys_name]"
+		if(string_to_test in destination_names)
+			destination_names.Remove(string_to_test)
+
+
+
+/datum/lore/organization/tsc/hephaestus
+	name = "Hephaestus Industries"
+	short_name = "Hephaestus"
+	acronym = "HI"
+	desc = "Hephaestus Industries is the largest supplier of arms, ammunition, and small military vehicles in Sol space. \
+	Hephaestus products have a reputation for reliability, and the corporation itself has a noted tendency to stay removed \
+	from corporate politics. They enforce their neutrality with the help of a fairly large asset-protection contingent which \
+	prevents any contracting polities from using their own materiel against them. SolGov itself is one of Hephastus' largest \
+	bulk contractors owing to the above factors."
+	history = ""
+	work = "arms manufacturer"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("HTV" = "freight", "HTV" = "munitions resupply")
+	destination_names = list(
+		"a SolGov dockyard on Luna"
+		)
+
+/datum/lore/organization/tsc/vey_med
+	name = "Vey Medical"
+	short_name = "Vey Med"
+	acronym = "VM"
+	desc = "Vey-Med is one of the newer TSCs on the block and is notable for being largely owned and operated by Skrell. \
+	Despite the suspicion and prejudice leveled at them for their alien origin, Vey-Med has obtained market dominance in \
+	the sale of medical equipment-- from surgical tools to large medical devices to the Odysseus trauma response mecha \
+	and everything in between. Their equipment tends to be top-of-the-line, most obviously shown by their incredibly \
+	human-like FBP designs. Vey's rise to stardom came from their introduction of ressurective cloning, although in \
+	recent years they've been forced to diversify as their patents expired and Nanotrasen-made medications became \
+	essential to modern cloning."
+	history = ""
+	work = "medical equipment supplier"
+	headquarters = ""
+	motto = ""
+
+	ship_prefixes = list("VTV" = "transportation", "VMV" = "medical resupply")
+	destination_names = list(
+	"a outpost on Jargon 4"
+	)
+
```

Comment:
/datum/lore/organization/tsc/einstein
	name = "Einstein Engines Incorporated"
	short_name = "Einstein Inc."
	acronym = "EEI"
	desc = "An Engineering firm specializing in alternative fuel-technologies for FTL travel, Einstein Engines is an up and coming player in the galactic FTL and energy markets. As their research into alternative FTL fuel threatens both Nanotrasen's relative stranglehold on plasma as well as The Syndicate's vested interest in the market, they are often the target of industrial sabotage by both Nanotrasen and The Syndicate. Most of their contracts are based outside of the Epsilon Eridani sector, and they are frequently commissioned by smaller firms to retrofit new and existing colonies, space stations, and outposts."
	history = ""
	work = "engineering firm specializing in engine technology"
	headquarters = "Jargon 4"
	motto = ""

	ship_prefixes = list("EE-T" = "transportation")
	destination_names = list()